src/bindings/python/make_binding.py

```diff
@@ -32,6 +36,7 @@ def process_header(f):
       s = header.read()
       s = re.sub(r'\\\n', '', s)
       s = re.sub(r'\/\*([\s\S]*?)\*\/', '', s)
+      s = re.sub(r',\s*\n', ', ', s, flags=re.MULTILINE)
```

Comment:
We've chatted about these over coffee before, and so I have a general idea what they are doing.  Unfortunately, I'm having a little difficulty reverse-enginnering the exact function of each.  I imagine that these will need some tweaking in the future as the codebase's headers change.  So I think it would be worth throwing a comment above each regex to specify what the intended target of the removal is and why it is being removed.

My guesses for each:

- The first one is removing the literal `\n`?  Not sure why.
- The second is removing block comments that are on a single line. I'm not sure why, I would assume cffi could handle that (or a pass through a pre-processor would take care of it).
- The third is collapsing multi-line statements into a single line. I assume to make detecting variadic functions easier.
- The fourth is removing the gcc __attribute__ keyword that I assume cffi chokes on.